SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/Link.h

```diff
@@ -323,9 +285,6 @@ class TLink : public BaseLink
     typedef typename TraitsContainer::T Container;
     typedef typename Container::const_iterator const_iterator;
     typedef typename Container::const_reverse_iterator const_reverse_iterator;
-    typedef LinkTraitsFindDest<OwnerType, DestType, ACTIVEFLAG(FLAG_DATALINK)> TraitsFindDest;
```

Comment:
Enjoy.

Reply:
Congratulating his own PR sounds a bit like people who like their own profile picture on FB ? 😆

Reply:
I think "Enjoy" is commonly said when you offer something to other that is enjoyable... like in pub the waiter says "Enjoy your drinks". 
So I don't think this is related to congratulation. 
But maybe my english is not good enough :)